packages/rollup-plugin-import-meta-assets/test/fixtures/dynamic-vars.js

```diff
@@ -0,0 +1,8 @@
+const one = 'one';
+// value of one could also be "two", bundler does not analyze that format
```

Comment:
Could we elaborate on how the variable `one` could have the value `"two"` here? It's not apparent to me at least.

Reply:
> You could write
> 
> ```
> const urls = ['one', 'two'].map( v => new URL(`./dynamic-assets/${one}.svg`, import.meta.url));
> ```

Ahh, could we use this snippet in the test fixture to make it obvious to readers?

Reply:
Yes!